packages/airnode-deployer/terraform/airnode/gcp/templates/apigateway.yaml.tpl

```diff
@@ -0,0 +1,54 @@
+swagger: "2.0"
```

Comment:
I guess this is for the open API file referenced in terraform, right?

Reply:
Yes.